### PR TITLE
pmd-google-checkstyle

### PR DESCRIPTION
**Pull request by gopal-mahajan**

### CODE REVIEW

**Comment by gopal-mahajan** [comment]
[PMD_Results_Ingestion_Server.pdf](https://github.com/datakaveri/iudx-data-ingestion-server/files/11331137/PMD_Results_Ingestion_Server.pdf)

[Checkstyle_Results_Ingestion_Server.pdf](https://github.com/datakaveri/iudx-data-ingestion-server/files/11331138/Checkstyle_Results_Ingestion_Server.pdf)

**Comment by kailash** [comment]
ok to test

**Comment by kailash** [comment]
retest this please

File: src/main/java/iudx/data/ingestion/server/authenticator/authorization/AdminAuthStrategy.java [hunk 1]
```diff
@@ -1,39 +1,29 @@
...
 public class AdminAuthStrategy implements AuthorizationStrategy {
-  private static final Logger LOGGER = LogManager.getLogger(AdminAuthStrategy.class);
-
   static Map<String, List<AuthorizationRequest>> AdminAuthorizationRules = new HashMap<>();
-  private final Api apis;
   private static volatile AdminAuthStrategy instance;
-  private AdminAuthStrategy(Api apis) {
-    this.apis=apis;
+
+  private AdminAuthStrategy() {
```
Comment (kailash) [inline src/main/java/iudx/data/ingestion/server/authenticator/authorization/AdminAuthStrategy.java:13]: Remove default Constructor

File: src/main/java/iudx/data/ingestion/server/authenticator/model/JwtData.java [hunk 2]
```diff
@@ -6,7 +6,7 @@
 @DataObject(generateConverter = true, publicConverter = false)
 public final class JwtData {
 
-  private String access_token;
+  private String accessToken;
```
Comment (kailash) [inline src/main/java/iudx/data/ingestion/server/authenticator/model/JwtData.java:9]: call `setAccessToken()` from `JwtData(JsonObject json)`  as done in RS [here](https://github.com/datakaveri/iudx-resource-server/pull/426/files#diff-b21ebe79a0d682898e2a4b849071d35fe264a4154823384bcc12a6cd0ee21f38R25).

**Review by kailash — CHANGES_REQUESTED** [review]

File: src/main/java/iudx/data/ingestion/server/authenticator/model/JwtData.java (hunk 2 above)
Comment (gopal-mahajan) [reply src/main/java/iudx/data/ingestion/server/authenticator/model/JwtData.java:9]: done

**Review by kailash — APPROVED** [review]

**Pull request merged**